helper_crates/vtable/src/vrc.rs

```diff
@@ -313,6 +313,11 @@ impl<VTable: VTableMetaDropInPlace + 'static, X> VWeak<VTable, X> {
             None
         }
     }
+
+    /// Returns true if the two VRc's point to the same allocation
+    pub fn ptr_eq(this: &Self, other: &Self) -> bool {
+        this.inner == other.inner
```

Comment:
This might be worth a unit test in the same file.

Reply:
In the same file is difficult because we need to have instances of things that inherit the right trait.
Maybe test_vrc.rs is better